### DISCUSSION TITLE
"Invalid link, valid link should contain tempPubKey" error shown without changing code

### DISCUSSION THREAD

**Discussion by ickw** [Web3Auth/Web3Auth#1236] [Help]
Hi there,

We've been developing our web application with Web3Auth for a while.
We use Web3Auth **Email authentication** feature for signing up/creating wallet for our application.

It's been working without major problems, but about from last week,
we suddenly get error shown on the window which opens when user click the verification link on email.
The error says "Invalid link, valid link should contain tempPubKey" (*Image attached),
but we have no idea why this starts happening because we didn't change our source code of our application.

This error happens irregularly as well, 
so it's very hard for us to reproduce this error to identify what the cause is.

Could you please helps us solving this problem?

Thanks.

<img width="647" alt="image001 (1)" src="https://user-images.githubusercontent.com/8847094/213918644-7ff01ee0-2803-4c6e-9f84-724a39577f96.png">

**Comment by arch1995** [comment]
Hi @ickw, 

We recently released a new version of the website but on some browsers, users are seeing an older version of the website because of the service worker implementation.

To resolve this, users can do a hard refresh which will solve the issue for them.
